sendemail.py

```diff
@@ -0,0 +1,16 @@
+from apscheduler.schedulers.blocking import BlockingScheduler
+import requests
+
+sched = BlockingScheduler()
+count=0
+
+@sched.scheduled_job('interval', minutes=1)
+def timed_job():
+    print('This job is run every one minutes.')
+    global count, sched
```

Comment:
Is `global` the only way? And why does `sched` need to be a global? It's not used in the code if I read correctly